ui/raidboss/data/06-ew/raid/p2n.ts

```diff
@@ -0,0 +1,178 @@
+import NetRegexes from '../../../../../resources/netregexes';
+import Outputs from '../../../../../resources/outputs';
+import { callOverlayHandler } from '../../../../../resources/overlay_plugin_api';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { PluginCombatantState } from '../../../../../types/event';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export interface Data extends RaidbossData {
+  bodyActor?: PluginCombatantState;
+}
+
+const triggerSet: TriggerSet<Data> = {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  zoneId: ZoneId.AsphodelosTheSecondCircle,
+  timelineFile: 'p2n.txt',
+  triggers: [
+    {
+      id: 'P2N Murky Depths',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '680F', source: 'Hippokampos', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P2N Doubled Impact',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '680E', source: 'Hippokampos' }),
+      response: Responses.sharedTankBuster(),
+    },
+    {
+      id: 'P2N Spoken Cataract',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: ['67F8', '67F7', '67F9'], source: 'Hippokampos' }),
```

Comment:
I'm a little surprised that there are only three here.  Is there a missing one for the other direction?

https://xivapi.com/search?indexes=action&string=Spoken%20Cataract&columns=Name,ID&pretty=true says there are five ability ids here, which I might expect if there's four directions for self-casting and one for damage (67FD).

Reply:
67FD is the one the body casts, I ask OverlayPlugin for the bosses position already so I don't care about the 67FD cast. (The heading value of the log line is wrong because of timing)

The head can never be aligned with the boss but only left/back/right hence 3 IDs.

I could add 67FD to the trigger but the execution would never yield any information and thus be ignored. (Think about the CPU cycles)

Reply:
> The head can never be aligned with the boss but only left/back/right hence 3 IDs.

Hmm, ok.  The fifth id is 0x67FE, but I don't see 0x67FE in my logs at all, so maybe that's unused/savage.

> (Think about the CPU cycles)

:joy: